frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/health-card.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card/card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import { HealthBody } from '@console/internal/components/dashboard/health-card/health-body';
+import { HealthItem } from '@console/internal/components/dashboard/health-card/health-item';
+import { HealthState } from '@console/internal/components/dashboard/health-card/states';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { STORAGE_HEALTH_QUERIES } from '../../../constants/queries';
+import { CEPH_HEALTHY, CEPH_DEGRADED, CEPH_ERROR, CEPH_UNKNOWN } from '../../../constants';
+
+const CephHealthStatus = {
+  0: {
```

Comment:
Why is this an object instead of an array?

Reply:
Agree with Sam, this should be an array, with individual elements of type `CephHealth`.

```ts
const cephHealthValues: CephHealth[] = [
  {
    state: HealthState.OK,
    message: CEPH_HEALTHY,
  },
  // the rest of values
];
```